phpstan.neon

```diff
@@ -10,6 +10,10 @@ parameters:
   excludePaths:
     analyseAndScan:
       - */social_demo/*
+      - tests/
+      - social.install
```

Comment:
I don't think we should add all of these to our exclusion from PHPstan analysis, also couldn't really understand from the commit / PR if there is a reason why you added all these?

Reply:
@ronaldtebrake These folders are getting included in `analyse` command of `phpstan`. So, when you try to generate a baseline, the errors get added too. That is why I am adding them to exclusion.

Reply:
Command: `../../../../vendor/bin/phpstan analyse .`

Output (sample) :
<img width="1728" alt="Screenshot 2022-02-24 at 5 34 43 PM" src="https://user-images.githubusercontent.com/8435994/155520583-1181f20f-e6c3-4d68-b6b0-a15a55d2b67d.png">

<img width="1728" alt="Screenshot 2022-02-24 at 5 35 10 PM" src="https://user-images.githubusercontent.com/8435994/155520626-24db4565-b0e3-4a34-a7d3-0e19d5a03772.png">